lib/puppet/provider/grafana_conn_validator/net_http.rb

```diff
@@ -0,0 +1,68 @@
+# See: #10295 for more details.
```

Comment:
it's not 100% clear to which bug tracker those ids relate. could you add the full URL?

Reply:
I'm not 100% sure either, I copied that from Puppetlab's PuppetDB years ago when I started using this validator pattern for things with APIs. It might also only apply to old Puppet so I'll have to see if Puppet 5 and 6 even suffer from whatever this linked to.

Reply:
I removed because the URL I get goes to crashed or broken https://projects.puppetlabs.com/issues/10295 , and rather not redirect people to broken sites.  I just left comment about why the load path changes are necessary